content/operate/kubernetes/deployment/helm.md

```diff
@@ -85,20 +84,77 @@ helm install <operator-name> redis/redis-enterprise-operator \
 
 ### Install with values file
 
-1. View configurable values with `helm show values <repo-name>/<chart-name>`.
+1. View configurable values with `helm show values <repo-name>/redis-enterprise-operator`.
 
 2. Create a YAML file to specify the values you want to configure.
 
 3. Install the chart with the `--values` option.
 
 ```sh
-helm install <operator-name> redis/redis-enterprise-operator \
+helm install <operator-name> <repo-name>/redis-enterprise-operator \
     --version <release-name> \
     --namespace <namespace-name> \
     --create-namespace \
     --values <path-to-values-file>
 ```
 
+## Migrate from a non-Helm installation
+
+To migrate an existing non-Helm installation of the Redis Enterprise operator to a Helm-based installation:
+
+1. [Upgrade]({{<relref "operate/kubernetes/upgrade">}}) your existing Redis Enterprise operator to match the version of the Helm chart you want to install. Use the same non-Helm method you used for the original installation.
+
+2. [Install](#install) the Helm chart adding the `--take-ownership` flag:
+
+   ```sh
+   helm install <release-name> <path-to-chart> --take-ownership
+   ```
+
+   - The `--take-ownership` flag is available with Helm versions 3.18 or later.
+   - This flag is only needed for the first installation of the chart. Subsequent upgrades don't require this flag.
+   - Use the `helm install` command, not `helm upgrade`.
+
+3. Delete the old `ValidatingWebhookConfiguration` object from the previous non-Helm installation:
+
+   ```sh
+   kubectl delete validatingwebhookconfiguration redis-enterprise-admission
+   ```
+
+   This step is only needed when the `admission.limitToNamespace` chart value is set to `true` (the default). In this case, the webhook object installed by the chart is named `redis-enterprise-admission-<namespace>`. If `admission.limitToNamespace` is set to `false`, the webhook installed by the chart is named `redis-enterprise-admission`, and the existing webhook object is reused.
+
+## Upgrade the chart
+
+To upgrade an existing Helm chart installation:
+
+```sh
+helm upgrade <release-name> <path-to-chart>
+```
+
+For example, to upgrade a chart with the release name `my-redis-enterprise` from the chart's root directory:
+
+```sh
+helm upgrade my-redis-enterprise .
+```
+
+You can also upgrade using the Redis Helm repository:
+
+```sh
+helm upgrade <release-name> <repo-name>/redis-enterprise-operator --version <chart-version>
+```
+
+To upgrade the chart on **OpenShift**, include the `openshift.mode=true` parameter:
+
+```sh
+helm upgrade <release-name> <path-to-chart> \
+     --set openshift.mode=true
+```
```

Comment:
I'm wondering whether this merits an explicit note like this.
Maybe just add the following sentence, like we do in the other install sections:
> To install with Openshift, add `--set openshift.mode=true`.